jcl/source/windows/JclWin32.pas

```diff
@@ -55,7 +55,11 @@
 {$I windowsonly.inc}
 
 {$MINENUMSIZE 4}
-{$ALIGN ON}
+{$IFDEF WIN64}
+  {$ALIGN 8}
+{$ELSE}
+  {$ALIGN 4}
+{$ENDIF}
```

Comment:
Why force an alignment that is not the default one from the compiler, which by the way is different for the CPU it targets.